server/services/interfaces/overviewPageService/OverviewPage.ts

```diff
@@ -1,61 +1,7 @@
-import FullStatus from '../../../data/interfaces/prisonApi/FullStatus'
-
-export default interface OverviewPage {
-  statuses: OverviewStatus[]
-  nonAssociationSummary: NonAssociationSummary
-  personalDetails: PersonalDetails
-  staffContacts: object
-  offencesOverview: {
-    mainOffenceDescription: string
-    fullStatus: FullStatus
-    imprisonmentStatusDescription: string
-    conditionalReleaseDate: string
-    confirmedReleaseDate: string
-  }
-  prisonName: string
-  staffRoles: string[]
-  isYouthPrisoner: boolean
-}
-
 export interface OverviewNonAssociation {
```

Comment:
If this is the only interface in the file, might make sense to rename the file to match?